tutorials/build_system/prerequisites.md

```diff
@@ -2,122 +2,201 @@
 title: Getting Started with Gradle
 ---
 
-Adding Gradle build support to an IntelliJ Platform Plugin requires a recent distribution of the Gradle build system and IntelliJ IDEA (Community or Ultimate). 
+Gradle is the preferred solution for creating IntelliJ Platform plugins.
+The IntelliJ IDEA Ultimate and Community editions bundle the necessary plugins to support Gradle-based development.
+These IntelliJ IDEA plugins are _Gradle_ and _Plugin DevKit_, which are enabled by default.
+To verify these plugins are installed and enabled, see the help section about [Managing Plugins](https://www.jetbrains.com/help/idea/managing-plugins.html).
 
-### 1.0 Download and Install IntelliJ IDEA
+* bullet list
+{:toc}
 
-Download and install either IntelliJ IDEA Ultimate or the IntelliJ IDEA Community Edition. 
 
-### 1.1 Ensure that 'Gradle' and 'Plugin DevKit' Plugins are Enabled
+## Creating a Gradle-Based IntelliJ Platform Plugin with New Project Wizard
+IntelliJ IDEA supports creating new Gradle-based IntelliJ Platform plugin projects using the [New Project Wizard](https://www.jetbrains.com/help/idea/gradle.html#project_create_gradle).
+The Wizard creates all the necessary project files based on a few template inputs.
 
-You can verify that the plugins are enabled by visiting **Settings \| Plugins**. 
+Before creating a new Gradle project, familiarize yourself with the IntelliJ IDEA help topic [Creating a new Gradle project](https://www.jetbrains.com/help/idea/getting-started-with-gradle.html#create_gradle_project), which is a tutorial for creating general Gradle projects in IntelliJ IDEA.
+This page emphasizes the steps in the process pertaining to Gradle-based IntelliJ Platform plugin projects.
 
-![Ensure the Gradle plugin is enabled](img/step0_gradle_enabled.png){:width="858px"}
+Launch the [New Project Wizard](https://www.jetbrains.com/help/idea/gradle.html#project_create_gradle).
+It guides you through the Gradle project creation process with four screens.
 
-### 1.2 Create a Plugin Project from Scratch
-
-IntelliJ IDEA supports automatically creating new plugin projects using Gradle, with all the necessary `build.gradle` setup performed automatically. 
-This can also be used to convert an existing plugin to Gradle, if Gradle is not able to convert the existing project - in this case, you need to copy over the sources to the new project. 
-
-To do so, create a new project in IntelliJ IDEA by opening **File \| New... \| Project**, and select Gradle from the dialog box. 
-In the "Additional Libraries and Frameworks" page, check "IntelliJ Platform Plugin".
+### New Project Configuration Screen
+On the first screen the type of project is configured:
+* From the _product category_ pane on the left, choose _Gradle_.
+* Specify the _Project SDK_.
+  Note that an [_IntelliJ Platform Plugin SDK_](/basics/getting_started/setting_up_environment.md#configuring-intellij-platform-sdk) can be used but is not required.
```

Comment:
I think there is no sense to use 'IntelliJ Platform Plugin SDK' for Gradle project, it'll just complicate things. I would write here that Java SDK should be used instead, as gradle-intellij-plugin will download required version of IntelliJ-based IDE automatically.

Reply:
Thanks, I will clarify.